include/deal.II/lac/la_vector.h

```diff
@@ -110,81 +110,88 @@ namespace LinearAlgebra
     /**
      * Multiply the entire vector by a fixed factor.
      */
-    virtual VectorSpaceVector<Number> &operator*= (const Number factor);
+    virtual Vector<Number> &operator*= (const Number factor) override;
```

Comment:
Any reason why you add the 'override' keyword here? Isn't it there for debugging purposes so you technically don't need it (since we don't use it elsewhere - it is C++11).